PublicGitArchive/pga-create/cmd/pga-create/indexer.go

```diff
@@ -13,13 +13,15 @@ import (
 )
 
 type indexCommand struct {
-	Output    string `short:"o" long:"output" default:"data/index.csv" description:"csv file path with the results"`
-	Debug     bool   `long:"debug" description:"show debug logs"`
-	LogFile   string `long:"logfile" description:"write logs to file"`
-	Limit     uint64 `long:"limit" description:"max number of repositories to process"`
-	Offset    uint64 `long:"offset" description:"skip initial n repositories"`
-	Workers   int    `long:"workers" description:"number of workers to use (defaults to number of CPUs)"`
-	ReposFile string `long:"repos-file" description:"path to a file with a repository per line, only those will be processed"`
+	Output       string `short:"o" long:"output" default:"data/index.csv" description:"csv file path with the results"`
+	Debug        bool   `long:"debug" description:"show debug logs"`
+	LogFile      string `long:"logfile" description:"write logs to file"`
+	Limit        uint64 `long:"limit" description:"max number of repositories to process"`
+	Offset       uint64 `long:"offset" description:"skip initial n repositories"`
+	Workers      int    `long:"workers" description:"number of workers to use (defaults to number of CPUs)"`
+	ReposFile    string `long:"repos-file" description:"path to a file with a repository per line, only those will be processed"`
+	Stars        string `short:"s" long:"stars" default:"data/stars.gz" description:"Input path for the file with the numbers of stars per repository."`
```

Comment:
To keep the description homogeneous it should start with lowercase letter and do not end with `.`. Same for `Repositories` option.